scene/gui/code_edit.cpp

```diff
@@ -1868,7 +1868,7 @@ bool CodeEdit::is_line_folded(int p_line) const {
 	return p_line + 1 < get_line_count() && !_is_line_hidden(p_line) && _is_line_hidden(p_line + 1);
 }
 
-TypedArray<int> CodeEdit::get_folded_lines() const {
+TypedArray<int> CodeEdit::get_folded_lines_bind() const {
```

Comment:
Is this an appropriate name? For these kinds of "middle-man" functions the naming is all over the place in the codebase.

Reply:
There are a few examples of such methods
https://github.com/godotengine/godot/blob/9283328fe70d27f148c403943a283d93a9785beb/core/math/rect2.h#L233-L234
https://github.com/godotengine/godot/blob/9283328fe70d27f148c403943a283d93a9785beb/core/object/worker_thread_pool.cpp#L399-L400
https://github.com/godotengine/godot/blob/9283328fe70d27f148c403943a283d93a9785beb/modules/websocket/websocket_peer.cpp#L95-L96